openfl/interface/aggregator.py

```diff
@@ -91,6 +92,28 @@ def start_(plan, authorized_cols, task_group):
         parsed_plan.config["assigner"]["settings"]["selected_task_group"] = task_group
         logger.info(f"Setting aggregator to assign: {task_group} task_group")
 
+        # Check if remote attestation is enabled in the plan configuration
+    if "network" in parsed_plan.config and "settings" in parsed_plan.config["network"]:
+        if parsed_plan.config["network"]["settings"].get("enable_remote_attestation", False):
```

Comment:
this would be equivalent to simply writing:
```python
if parsed_plan.config.get("network", {}).get("settings", {}).get("enable_remote_attestation", False):
```

Reply:
This check is added as unit tests are sending Mock Object which doesn't have config or is an instance of PLAN. Hence added the check.